database/src/main/kotlin/no/nav/su/se/bakover/database/sak/SakPostgresRepo.kt

```diff
@@ -47,6 +48,16 @@ internal class SakPostgresRepo(
         }
     }
 
+    override fun hentAlleSaker(): List<Sak> {
+        return dataSource.withSession { session ->
+            """
+                select * from sak
```

Comment:
Spennede å se hvor langt denne skalerer, tipper det ikke holder så veldig lenge. Tenker dog at det er greit nok for en første versjon, men vi må/bør sannsynligvis se mer på dette når vi skal tilby søkefunksjonalitet uansett.

Dersom det tar veldig lang tid kan vi vurderer å "gjemme" siden slik at man aktivt må oppsøke den inntil vi har fått fikset problemet.